Crash when Artifact size is larger than Integer max value

File sizes can easily exceed 32 bit `Integer` range, which results in a crash.

```
Caused by: com.fasterxml.jackson.databind.JsonMappingException: Numeric value (6495215233) out of range of int (-2147483648 - 2147483647)
 at [Source: (org.glassfish.jersey.client.internal.HttpUrlConnector$2); line: 1, column: 2082] (through reference chain: java.util.ArrayList[0]->org.gitlab4j.api.models.Job["artifacts"]->java.util.ArrayList[0]->org.gitlab4j.api.models.Artifact["size"])
	at com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:392)
	at com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:351)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.wrapAndThrow(BeanDeserializerBase.java:1821)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:315)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:176)
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer._deserializeFromArray(CollectionDeserializer.java:355)
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:244)
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:28)
	at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:129)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:313)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:176)
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer._deserializeFromArray(CollectionDeserializer.java:355)
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:244)
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:28)
	at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:322)
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4675)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3683)
	at org.gitlab4j.api.Pager.<init>(Pager.java:95)
	... 10 common frames omitted
Caused by: com.fasterxml.jackson.core.exc.InputCoercionException: Numeric value (6495215233) out of range of int (-2147483648 - 2147483647)
 at [Source: (org.glassfish.jersey.client.internal.HttpUrlConnector$2); line: 1, column: 2082]
	at com.fasterxml.jackson.core.base.ParserMinimalBase._reportInputCoercion(ParserMinimalBase.java:623)
	at com.fasterxml.jackson.core.base.ParserMinimalBase.reportOverflowInt(ParserMinimalBase.java:592)
	at com.fasterxml.jackson.core.base.ParserBase.convertNumberToInt(ParserBase.java:946)
	at com.fasterxml.jackson.core.base.ParserBase._parseIntValue(ParserBase.java:859)
	at com.fasterxml.jackson.core.base.ParserBase.getIntValue(ParserBase.java:689)
	at com.fasterxml.jackson.databind.deser.std.NumberDeserializers$IntegerDeserializer.deserialize(NumberDeserializers.java:526)
	at com.fasterxml.jackson.databind.deser.std.NumberDeserializers$IntegerDeserializer.deserialize(NumberDeserializers.java:506)
	at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:129)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:313)
	... 24 common frames omitted
```

--- a/src/main/java/org/gitlab4j/api/models/Artifact.java
+++ b/src/main/java/org/gitlab4j/api/models/Artifact.java
@@ -30,7 +30,7 @@ public class Artifact {
     }
 
     private FileType fileType;
-    private Integer size;
+    private Long size;
     private String filename;
     private String fileFormat;
 
@@ -42,11 +42,11 @@ public class Artifact {
         this.fileType = fileType;
     }
 
-    public Integer getSize() {
+    public Long getSize() {
         return size;
     }
 
-    public void setSize(Integer size) {
+    public void setSize(Long size) {
         this.size = size;
     }
 

--- a/src/test/resources/org/gitlab4j/api/job.json
+++ b/src/test/resources/org/gitlab4j/api/job.json
@@ -15,6 +15,7 @@
     },
     "artifacts": [
       {"file_type": "archive", "size": 1000, "filename": "artifacts.zip", "file_format": "zip"},
+      {"file_type": "archive", "size": 9223372036854775807 , "filename": "long.max_value.sized.zip", "file_format": "zip"},
       {"file_type": "metadata", "size": 186, "filename": "metadata.gz", "file_format": "gzip"},
       {"file_type": "trace", "size": 1500, "filename": "job.log", "file_format": "raw"},
       {"file_type": "junit", "size": 750, "filename": "junit.xml.gz", "file_format": "gzip"}

